keyboards/printedpad/rules.mk

```diff
@@ -0,0 +1,8 @@
+ifeq ($(KEYMAP), noah)
+	SRC += oled2.c
+else
+	SRC += oled1.c
+endif
+
+OLED_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
The `oled2.c` file should be moved to `keymaps/noah` and included there, or the code just placed in the `keymap.c` file. Then, the code in `oled1.c` file should be moved to `printedpad.c`.
```suggestion
# This file intentionally left blank
```

Reply:
I removed everything except SRC += oled.c in order to keep the oled code contained. This let me put oled2.c at the keymap by having both files named oled.c.